class.jetpack-cli.php

```diff
@@ -559,6 +559,189 @@ public function options( $args, $assoc_args ) {
+					) {
+						$modules[ 'users' ] = 'initial';
+					} elseif ( isset( $assoc_args[ $module_name ] ) ) {
+						$ids = explode( ',', $assoc_args[ $module_name ] );
+						if ( count( $ids ) > 0 ) {
+							$modules[ $module_name ] = $ids;
+						}
+					}
+				}
+
+				if ( empty( $modules ) ) {
+					$modules = null;
+				}
+
+				// Kick off a full sync
+				if ( Jetpack_Sync_Actions::do_full_sync( $modules ) ) {
+					if ( $modules ) {
+						WP_CLI::log( sprintf( __( 'Initialized a new full sync with modules: ', 'jetpack' ), join( ', ', $modules ) ) );
+					} else {
+						WP_CLI::log( __( 'Initialized a new full sync', 'jetpack' ) );
+					}
+				} else {
+
+					// Reset sync settings to original.
+					Jetpack_Sync_Settings::update_settings( $original_settings );
+
+					if ( $modules ) {
+						WP_CLI::error( sprintf( __( 'Could not start a new full sync with modules: %s', 'jetpack' ), join( ', ', $modules ) ) );
+					} else {
+						WP_CLI::error( __( 'Could not start a new full sync', 'jetpack' ) );
+					}
+				}
+
+				// Keep sending to WPCOM until there's nothing to send
+				$i = 1;
+				do {
+					$result = Jetpack_Sync_Actions::$sender->do_full_sync();
+					if ( $result ) {
+						if ( 1 == $i++ ) {
+							WP_CLI::log( __( 'Sent data to WordPress.com', 'jetpack' ) );
+						} else {
+							WP_CLI::log( __( 'Sent more data to WordPress.com', 'jetpack' ) );
+						}
+					}
+				} while ( $result );
+
+				// Reset sync settings to original.
+				Jetpack_Sync_Settings::update_settings( $original_settings );
+
+				WP_CLI::success( __( 'Finished syncing to WordPress.com', 'jetpack' ) );
+				break;
+		}
+	}
+
+	/**
+	 * List the contents of a specific Jetpack sync queue.
+	 *
+	 * ## OPTIONS
+	 *
+	 * peek : List the 100 front-most items on the queue.
+	 *
+	 * ## EXAMPLES
+	 *
+	 * wp jetpack sync_queue full_sync peek
+	 *
+	 * @synopsis <incremental|full_sync> <peek>
+	 */
+	public function sync_queue( $args, $assoc_args ) {
+		if ( ! Jetpack_Sync_Actions::sync_allowed() ) {
+			WP_CLI::error( __( 'Jetpack sync is not currently allowed for this site.', 'jetpack' ) );
+		}
+
+		$queue_name = isset( $args[0] ) ? $args[0] : 'sync';
+		$action = isset( $args[1] ) ? $args[1] : 'peek';
+
+		// We map the queue name that way we can support more friendly queue names in the commands, but still use
+		// the queue name that the code expects.
+		$queue_name_map = $allowed_queues = array(
+			'incremental' => 'sync',
+			'full'        => 'full_sync',
+		);
+		$mapped_queue_name = isset( $queue_name_map[ $queue_name ] ) ? $queue_name_map[ $queue_name ] : $queue_name;
+
+		switch( $action ) {
+			case 'peek':
+				require_once JETPACK__PLUGIN_DIR . 'sync/class.jetpack-sync-queue.php';
+				$queue = new Jetpack_Sync_Queue( $mapped_queue_name );
+				$items = $queue->peek( 100 );
+
+				if ( empty( $items ) ) {
+					WP_CLI::log( sprintf( __( 'Nothing is in the %s queue', 'jetpack' ), $queue_name  ) );
```

Comment:
Please don't create translatable strings like that. `incremental` and `full` can be integrated like that in English but this doesn't work for other languages, also translators need to know what the placeholder is (hint: `translators:` comment :) ). A string that works for a construct like you were using: `Nothing is in the queue: %s`